operations/kubernetes/Repopulate_Data_in_etcd_Clusters_When_Rebuilding_Them.md

```diff
@@ -25,8 +25,8 @@ A etcd cluster was rebuilt. See [Rebuild Unhealthy etcd Clusters](Rebuild_Unheal
 
     Boot preparation information for other product streams can be found in the following locations:
 
-    -   UANs: refer to the UAN product stream repository and search for the "PREPARE UAN BOOT SESSION TEMPLATES" header in the "Install and Configure UANs" procedure.
-    -   Cray Operating System \(COS\): refer to the COS product stream repository and search for the "Create a Boot Session Template" header in the "Install or Upgrade COS" procedure.
+    -   UANs: Refer to the UAN product stream repository and search for the "PREPARE UAN BOOT SESSION TEMPLATES" header in the "Install and Configure UANs" procedure.
+    -   Cray Operating System \(COS\): Refer to the "Create a Boot Session Template" header in the "Install or Upgrade COS" procedure in the COS product stream documentation.
```

Comment:
Change to

    -   Cray Operating System \(COS\): Refer to the "Create a Boot Session Template" header in the "Boot COS" procedure in the COS product stream documentation.